doc/manual/citing_aspect.bib

```diff
@@ -1138,7 +1138,7 @@ @article{STEIN2022229276
 
 @article{https://doi.org/10.1029/2021JB023244,
 author = {Zha, Caicai and Lin, Jian and Zhou, Zhiyuan and Xu, Min and Zhang, Xubo},
-title = {Effects of Hotspot-Induced Long-Wavelength Mantle Melting Variations on Magmatic Segmentation at the Reykjanes Ridge: Insights From 3D Geodynamic Modeling},
+title = {Effects of Hotspot-Induced Long-Wavelength Mantle Melting Variations on Magmatic Segmentation at the {R}eykjanes Ridge: Insights From 3D Geodynamic Modeling},
```

Comment:
How do you decide what letters to force uppercase? Ridge is part of the name, shouldn't it also be uppercase? What about 3D and Insights?

Reply:
I do not decide. I take whatever is the "official" export for the publisher is and fix any obvious translation issues of special characters.

Reply:
I can't say that I had a philosophy, I just wanted to improve things :-) If you tell me what you want, I'll fix it.

It probably doesn't matter much here because we don't use bibtex for these entries, but jabref. I don't believe that jabref actually translates to lower-case, though I may be wrong.

Reply:
yeah, jabref keeps the capitalization as is it seems. All the same to me. :-)